packages/browser/src/builder/BacktraceClientBuilder.ts

```diff
@@ -16,27 +23,33 @@ import { SpiderMonkeyStackTraceConverter } from '../converters/SpiderMonkeyStack
 import { getEngine } from '../engineDetector';
 
 export class BacktraceClientBuilder extends BacktraceCoreClientBuilder<BacktraceClient> {
-    constructor(protected readonly options: BacktraceConfiguration) {
-        super(
-            new BacktraceBrowserRequestHandler(options),
-            [
-                new UserAgentAttributeProvider(),
-                new WebsiteAttributeProvider(),
-                new WindowAttributeProvider(),
-                new UserIdentifierAttributeProvider(options),
-                new ApplicationInformationAttributeProvider(options),
-            ],
-            [new WebRequestEventSubscriber(), new DocumentEventSubscriber(), new HistoryEventSubscriber()],
-        );
+    constructor(
+        protected readonly options: BacktraceConfiguration,
+        attributeProviders: BacktraceAttributeProvider[] = [
+            new UserAgentAttributeProvider(),
+            new WebsiteAttributeProvider(),
+            new WindowAttributeProvider(),
+            new UserIdentifierAttributeProvider(options),
+            new ApplicationInformationAttributeProvider(options),
+        ],
+        breadcrumbSubscribers: BreadcrumbsEventSubscriber[] = [
```

Comment:
May want to make this (and other references) plural.

Reply:
it is plural, am I missing something?

Reply:
I was thinking breadcrumbSubscribers -> breadcrumb**s**Subscribers

Reply:
Ah OK got it